docs/operator/deploy.md

```diff
@@ -6,143 +6,114 @@ sidebarDepth: 2
 
 # Deployment guide
 
-This document explains in details the workflow to have a working group of drand
-nodes generate randomness. On a high-level, the workflow looks like this:
+This document explains the workflow to have a working group of drand nodes generate randomness. There are three main sections to this guide:
 
-- **Setup**: generation of individual long-term key pair and the group file and
-  starting the drand daemon.
-- **Distributed Key Generation**: each drand node collectively participates in
-  the DKG.
-- **Randomness Generation**: the randomness beacon automatically starts as soon
-  as the DKG protocol is finished.
+- generate the long-term key pairs and the group file.
+- start the daemons.
+- have each node collectively particiate in the distributed key generation (DKG).
+- genearte randomness.
 
 ## Setup
 
 The setup process for a drand node consists of the following steps:
 
-1. Generate the long-term key pair for each node
-2. Each node starts their daemon
-3. Leader starts the command as a coordinator & every participant connects to the
-   coordinator to setup the network
+1. Generate the long-term key pair for each node.
+1. Start the drand daemon on each node.
+1. The leader starts the command as a coordinator & every participant connects to the coordinator to setup the network.
 
-This document explains how to do the setup with the drand binary itself. drand
-also offers a docker image to run the setup. You can find the information for
-running with docker [here](https://github.com/drand/drand/blob/master/docker/README.md).
+This document explains how to do the setup with the drand binary itself. If you want to install drand using Docker, follow the [Docker instructions instead](/operate/docker).
 
 ### Long-term key
 
-To generate the long-term key pair `drand_id.{secret,public}` of the drand
-daemon, execute
+Each drand node needs a public and secret key to interact with the rest of the network. To generate these keys run `drand generate-keypair` followed by the address of your node:
```

Comment:
Link to CLI docs for `generate-keypair` please!

Reply:
Done!